src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/task.py

```diff
@@ -282,10 +291,16 @@ def acr_task_update(cmd,  # pylint: disable=too-many-locals
     if task.trigger:
         source_triggers = task.trigger.source_triggers
         base_image_trigger = task.trigger.base_image_trigger
-        if commit_trigger_enabled or source_triggers is not None:
-            status = None
+        if (commit_trigger_enabled or pull_request_trigger_enabled or source_triggers) is not None:
```

Comment:
nit: Do we need `is not None` here?

Reply:
Thank you, please see the most recent push.